src/ray/protobuf/common.proto

```diff
@@ -196,6 +196,73 @@ message RayException {
   string formatted_exception_string = 3;
 }
 
+message ActorDeathCause {
+  oneof context {
+    CreationTaskFailureContext creation_task_failure_context = 1;
```

Comment:
```suggestion
    RayException creation_task_failure = 1;
```

Reply:
Hmm I prefer the current style. It is more consistent with other oneof (this easier to understand) and easier to add more metadata if needed in the future with minimal changes

Reply:
It's adding a wrapping layer that's hard to read, kind of over engineered imo.

Reply:
Hmm sure. In this case, what about just using the consistent name (but change the type to avoid nested message)? 

```
RayException  creation_task_failure_context
```

Reply:
Addressed